app/Http/Controllers/Admin/UserController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Requests\Admin\UserRequest;
+use HotelBooking\AdminUser;
+use Auth;
+
+class UserController extends AdminBaseController
+{
+      /**
+       * Show view edit profile Admin User
+       * @return Response
+       */
+      public function getEditProfile()
+      {
+        return view('admin.edit_profile');
+      }
+
+      /**
+       * [update Profile Admin User]
+       * @param  AdminUserRequest $request [description]
+       * @return Response
+       */
+      public function putEditProfile(UserRequest $request)
+      {
+        $new_password = $request->new_password;
+        $old_password = $request->old_password;
+        if(!password_verify($old_password,Auth::admin()->get()->password))
```

Comment:
Use Laravel way to verify password [http://laravel.com/docs/5.0/hashing#basic-usage](http://laravel.com/docs/5.0/hashing#basic-usage)